fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -2917,7 +2917,25 @@ class DDTeamCollectionImpl {
 		if (res.size() > 0) {
 			// SOMEDAY: support wiggle multiple SS at once
 			ASSERT(!self->wigglingId.present()); // only single process wiggle is allowed
-			self->wigglingId = res.begin()->first;
+
+			state Optional<Value> localityKey;
+			state Optional<Value> localityValue;
+			if (self->configuration.perpetualStorageWiggleLocality != "0") {
+				ParsePerpetualStorageWiggleLocality(
```

Comment:
I think they don't need to be state variables.
```suggestion
			Optional<Value> localityKey;
			Optional<Value> localityValue;
			if (self->configuration.perpetualStorageWiggleLocality != "0") {
				ParsePerpetualStorageWiggleLocality(
```

Reply:
Done